homeassistant/components/imeon_inverter/quality_scale.yaml

```
@@ -0,0 +1,24 @@
+rules:
+  # Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
+  test-before-setup: todo
```

```suggestion
  test-before-setup: done
```
will be handled through `async_config_entry_first_refresh`